tests/components/rfxtrx/test_switch.py

```diff
@@ -219,16 +127,12 @@ def test_one_switch(self):
         entity.turn_off()
         assert not entity.is_on
 
-        entity_id = rfxtrx_core.RFX_DEVICES["213c7f216"].entity_id
-        entity_hass = self.hass.states.get(entity_id)
-        assert "Test" == entity_hass.name
-        assert "off" == entity_hass.state
+        assert "Test" == entity.name
+        assert "off" == entity.state
         entity.turn_on()
-        entity_hass = self.hass.states.get(entity_id)
-        assert "on" == entity_hass.state
+        assert "on" == entity.state
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)